dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1060,9 +1056,13 @@ void TeachingTip::StartExpandToOpen()
         if (m_beakOcclusionGrid)
         {
             m_beakOcclusionGrid.get().StartAnimation(m_expandAnimation.get());
-            m_beakOcclusionGrid.get().StartAnimation(m_expandElevationAnimation.get());
             m_isExpandAnimationPlaying = true;
         }
+        if (m_contentRootGrid)
+        {
+            m_contentRootGrid.get().StartAnimation(m_expandElevationAnimation.get());
+            m_isContractAnimationPlaying = true;
```

Comment:
Copy/paste issue?

Reply:
But same feedback from the other function -- I think this value should just be set after all 3 if statements.

Reply:
Same response..
These are all template parts, if we don't have any of them we wont have started any animations so the flag would need to be false.